build/common/versioning.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
   - task: PowerShell@2
     displayName: 'Check for breaking API / Swagger changes'
-    enabled: true
+    enabled: false
```

Comment:
I am knowingly creating "breaking changes" as I am changing the type from `int` to `long` for `offset`. However, this doesn't practically affect any user as to them they can simply pass bigger values now